environment

```diff
@@ -1,4 +1,4 @@
-export PATH=%MINC_TOOLKIT_DIR%/bin:/data/%PROJECT%/bin/mri:/data/%PROJECT%/bin/mri/uploadNeuroDB:/data/%PROJECT%/bin/mri/dicom-archive:$PATH
+export PATH=%MINC_TOOLKIT_DIR%/bin:/data/%PROJECT%/bin/mri:/data/%PROJECT%/bin/mri/uploadNeuroDB:/data/%PROJECT%/bin/mri/uploadNeuroDB/bin:/data/%PROJECT%/bin/mri/dicom-archive:$PATH
```

Comment:
do you need to add the `bin` directory? just before you have the `uploadNeuroDB`, wouldn't this encompass the bin directory within?
just curious.
I ask but I know without the change the insertion would fail, but I am just curious!

another detail, in the description of this PR n top, you say this path needs to be added to the beginning, but you put it towards the end here. Does it matter?